src/main.rs

```diff
@@ -11,15 +11,17 @@ mod iot;
 mod other;
 mod sdk;
 mod server;
+mod twim_config;
 
 fn main() {
     // 1. Open and parse the toml containing all data
     const PROJECT_DATA_PATH: &str = "./projects.toml";
-    let mut file = File::open(PROJECT_DATA_PATH)
-        .expect("Unable to open file");
+    const TWIM_CONFIG_PATH: &str = "../twim-config/config.toml";
+    let mut projects_file = File::open(PROJECT_DATA_PATH)
+        .expect("Unable to open projects data file");
     let mut template = String::new();
-    file.read_to_string(&mut template)
-        .expect("Unable to read file");
+    projects_file.read_to_string(&mut template)
+        .expect("Unable to read twim config file");
```

Comment:
Do I need this template thing here?

Reply:
No, there is `fs::read_to_string` that does the same `File::open`, declaring a mutable string + `read_to_string`ing into that. There's also `fs::read` which I would prefer here (in combination w/ `toml::from_slice` instead of `toml::from_string`), to delay UTF-8 validation until TOML parsing (rather than file reading).